src/v/coproc/tests/CMakeLists.txt

```diff
@@ -35,6 +36,15 @@ rp_test(
   LABELS coproc
   )
 
+rp_test(
+  UNIT_TEST
+  BINARY_NAME wasm_event_tests
+  SOURCES wasm_event_tests.cc
+  DEFINITIONS BOOST_TEST_DYN_LINK
+  LIBRARIES v::seastar_testing_main v::coproc_test_utils
```

Comment:
can you combine most of the coproc tests into 1 large test?  - it makes linking faster. 

we prefer one target binary if the tests allow.